changelog/release-schedule.dd

```diff
@@ -0,0 +1,46 @@
+Ddoc
+
+$(D_S $(TITLE),
+
+$(UL
+    $(LI New release are published every $(I two) months, on the first day of every uneven month.)
+    $(LI Two weeks before a new release `master` is merged into `stable` and a first beta is released.)
+    $(LI Point releases are published unscheduled when important issues or regressions get fixed.)
+)
+
+$(P The release schedule for 2018 is as follows:)
+
+    $(DIVC release-schedule,
+        $(TABLE
+            $(MINOR_RELEASE 2018-03-01, 2.079.0)
+            $(BETA_RELEASE 2018-04-15, 2.080.0)
+            $(MINOR_RELEASE 2018-05-01, 2.080.0)
+            $(BETA_RELEASE 2018-06-15, 2.081.0)
+            $(MINOR_RELEASE 2018-07-01, 2.081.0)
+            $(BETA_RELEASE 2018-08-15, 2.082.0)
+            $(MINOR_RELEASE 2018-09-01, 2.082.0)
+            $(BETA_RELEASE 2018-10-15, 2.083.0)
+            $(MINOR_RELEASE 2018-11-01, 2.083.0)
+            $(BETA_RELEASE 2018-12-15, 2.084.0)
+            $(MINOR_RELEASE 2019-01-01, 2.084.0)
+        )
+    )
+)
+
+Macros:
+    TITLE=D release schedule
+    MINOR_RELEASE=$(ROW minor, $1, $2, minor release)
```

Comment:
I wonder if "major" release would be more appropriate, especially given how many new features 2.079.0 comes with. Major is also closer to the SemVer terminology, since deprecations and removal of features is allowed only in major versions.

Reply:
I wondered about that too, but major would mean that we bump from 2 to 3.
Anyhow the entire "releases aren't SemVer" is a long-standing issue though I have hopes that it will be finally fixed this year.
We could also use a non-semver word like "new feature release"?

Reply:
With the current version scheme, a bump from 2 to 3 would be mean a brand new language (like D1 -> D2), so I still believe that "major" release would be more true to the name.